package.json

```diff
@@ -11,7 +11,7 @@
     "build:ts": "tsc -p src/ --pretty",
     "build:next": "next build",
     "build:watch": "npm run build:ts -- -w",
-    "lint": "eslint \"src/**/*.ts\" --max-warnings 0",
+    "lint": "eslint --ext={js,jsx,ts,tsx} src --max-warnings 0",
```

Comment:
Out of curiosity do we need to have the `ext` flag if the extensions are also listed on line 31 of the `.eslintrc.js` file? If we can remove the flag that would be great, then we have them in one place. If it doesn't work by removing the flag though that's fine.

Reply:
Ok, I fixed the flag to be a string. From my understanding, the `.eslintrc.js` is what the ide uses to know the eslint rules, and anything in `package.json` is used for building and compiling but I may be wrong.

Reply:
Sounds good 👍